examples/site.yml

```diff
@@ -3,13 +3,12 @@
 
   pre_tasks:
     - name: Update APT cache
-      apt: update_cache=yes
+      apt:
+        update_cache: yes
 
   roles:
-    - { role: "azavea.python-security", when: 'ansible_python_version | version_compare("2.7.9", "<")' }
```

Comment:
No longer necessary because the version of Python 2.x on Xenial is greater than 2.7.9.